index.html

```diff
@@ -39,43 +42,102 @@ <h3 class="font-bold mb-2 text-xl">
+              </a>
+            </div>
+          </article>
 
-        <hr aria-hidden="true" class="my-6" />
+          <article class="w-full md:w-2/4 mb-3">
+            <div
+              class="mr-3 rounded bg-white shadow-md hover:shadow-lg transition group"
+            >
+              <a href="#" class="block p-3">
+                <h3
+                  class="font-bold mb-2 text-lg underline group-hover:text-red-400"
+                >
+                  #4 &ndash; ???
+                </h3>
+                <footer>
+                  <p class="mb-2">
+                    <span class="sr-only">Julkaistu</span>
+                    <time datetime="2021-03-27">27.3.2021</time>
+                  </p>
+                </footer>
+                <p>Jaksokuvaus...</p>
+              </a>
+            </div>
+          </article>
+          <article class="w-full md:w-2/4 mb-3">
+            <div
+              class="mr-3 rounded bg-white shadow-md hover:shadow-lg transition group"
+            >
+              <a href="#" class="block p-3">
+                <h3
+                  class="font-bold mb-2 text-lg underline group-hover:text-red-400"
+                >
+                  #4 &ndash; ???
+                </h3>
+                <footer>
+                  <p class="mb-2">
+                    <span class="sr-only">Julkaistu</span>
+                    <time datetime="2021-03-27">27.3.2021</time>
+                  </p>
+                </footer>
+                <p>Jaksokuvaus...</p>
+              </a>
+            </div>
+          </article>
 
-        <article class="-m-2 p-2 rounded hover:bg-yellow-100">
-          <a href="#">
-            <h3 class="font-bold mb-2 text-lg underline">#1 &ndash; Intro</h3>
-            <footer>
-              <p class="mb-2">
-                <span class="sr-only">Julkaistu</span>
-                <time datetime="2021-03-27">27.3.2021</time>
-              </p>
-            </footer>
-            <p>Jaksokuvaus...</p>
-          </a>
-        </article>
+          <article class="w-full md:w-2/4 mb-3">
+            <div
+              class="mr-3 rounded bg-white shadow-md hover:shadow-lg transition group"
+            >
+              <a href="#" class="block p-3">
+                <h3
+                  class="font-bold mb-2 text-lg underline group-hover:text-red-400"
+                >
+                  #4 &ndash; ???
+                </h3>
+                <footer>
+                  <p class="mb-2">
+                    <span class="sr-only">Julkaistu</span>
+                    <time datetime="2021-03-27">27.3.2021</time>
+                  </p>
+                </footer>
+                <p>Jaksokuvaus...</p>
+              </a>
+            </div>
+          </article>
+        </div>
       </main>
 
-      <footer class="border-t mt-8 pt-8">&copy; Mario Bros. 2021</footer>
+      <footer class="border-t mt-8 pt-8">&copy; Koodikrapula 2021</footer>
```

Comment:
Footer should have padding-bottom as well but we can add it later. Plus maybe the text could be lighter?